test/ParseExceptionTest.cpp

```diff
@@ -33,18 +36,18 @@ TEST(ParseException, MetadataGeneration) {
   // A is not a valid argument for select.
   expectParseExceptionWithMetadata(
       "SELECT A ?a WHERE { ?a ?b ?c }",
-      {{"select   A ?a WHERE { ?a ?b ?c }", 9, 9, 1, 9}});
+      {{"SELECT A ?a WHERE { ?a ?b ?c }", 7, 7, 1, 7}});
```

Comment:
Is the `The ANTLR Parser currently...` comment now outdated and can be finally removed? The actual query now looks completely sane.

Reply:
You are right that comment is now outdated.